### DISCUSSION TITLE
Perfect segmentation

### DISCUSSION THREAD

**Discussion by feersantana5** [isl-org/Open3D#3697] [Q&A] [answered]
I have done this tutorial https://towardsdatascience.com/how-to-automate-3d-point-cloud-segmentation-and-clustering-with-python-343c9039e4f5 to get better performance using DBSCAN method. However with my own pics I can't get a perfect segmentation as this man does. 

Using only DBSCAN method i get:
<img width="148" alt="image" src="https://user-images.githubusercontent.com/61494475/124538080-f1a78f80-de12-11eb-80e1-5dddf111998d.png">

After the tutorial I get:
![image](https://user-images.githubusercontent.com/61494475/124576766-87a7de00-de44-11eb-89f5-53f25e7f8f42.png)

I would like to be able to extract the chair using segmentation. Any trick? or parameter I should change?

**Comment by ssheorey** [comment, marked as answer]
Hey, DBSCAN is sensitive to parameter settings that depend on your scene:
`min_pts` and `eps`
http://www.open3d.org/docs/latest/python_api/open3d.geometry.PointCloud.html#open3d.geometry.PointCloud.cluster_dbscan

Please check the DBSCAN paper or the [Wikipedia article](https://en.wikipedia.org/wiki/DBSCAN) for help on setting these parameters.
